docs/competition/team-support/battery-charging.md

```diff
@@ -36,15 +36,21 @@ Teams will bring discharged batteries to the Battery Charging station and requir
 
 ### Charging a discharged battery
 
-All of the battery chargers will be set-up and ready to use. Before a battery is connected the screen must display `LiPo BALANCE 2.2A 11.1V(3S)`. If it displays anything else then please do not use it and ask [Helpdesk](/competition/team-support/helpdesk) to summon Rich Barlow. Batteries are charged as follows:
+All of the battery chargers will be set-up and ready to use. Before a battery is connected the screen must display `LiPo BALANCE 2.2A 11.1V(3S)`. If it displays anything else then please do not use it and ask [Helpdesk](/competition/team-support/helpdesk) to summon the head helpdesker. Batteries are charged as follows:
```

Comment:
This change will conflict with #21.

Separately, is the head helpdesker really the right person to summon here?

Reply:
I'm just going to revert this file and let #21 sort it